recipes/weasyprint/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "WeasyPrint" %}
+{% set version = "v47" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Kozea/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 707b3b54c6552c99f67682178316b653ed229b03ffb17158f7374e99829338eb
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
This could be `noarch: python`

Reply:
It has actually been changed because `noarch: python` is forbidden when some platforms are skipped (and Windows is).

Reply:
Oh right, good point. Thanks